### PR TITLE
Make Flutter frontend server Null safe.

### PR DESCRIPTION
**Pull request by a-siva**
Make flutter frontend server null safe.
Merging of this PR needs to be coordinated with @CaseyHillers

### CODE REVIEW

**Review by CaseyHillers — APPROVED** [review]
LGTM from the roll side. I'll re-add the g3fix once this is merged.

Please get a LGTM from the engine side as well.

**Review by jason-simmons — APPROVED** [review]

**Pull request merged**
